vagrant-kubernetes-istio/install_prereqs_debian.sh

```diff
@@ -41,8 +42,26 @@ if [ $? -ne 0 ]; then
       exit 1
     fi
     sudo apt-get install -f
+    # Check for more recent version and update
+    sudo apt-get install virtualbox
+    if [ $? -ne 0 ]; then
+      echo "Looks like virtual box update failed. Please try manually. "
+      exit 1
+    else
+      echo "virtual box install done!"
+    fi
+else
+    echo "Looks like virtual is installed. Checking if it can be upgraded."
+    sudo apt-get install virtualbox
```

Comment:
To upgrade virtualbox, do we need "sudo apt-get update" before "sudo apt-get install virtualbox"?

Reply:
This is not necessary, VBoxManage -v prints the virtualbox version, maybe it would be helpful to have this version information in output.